packages/front-end/components/EventWebHooks/EventWebHookAddEditModal/EventWebHookAddEditModal.tsx

```diff
@@ -232,7 +233,11 @@ const EventWebHookAddEditSettings = ({
       <div className="mt-4">
         <Field
           label={<b>Endpoint URL</b>}
-          placeholder="https://example.com/growthbook-webhook"
+          placeholder={
+            selectedPayloadType === "datadogEvent"
+              ? "https://api.datadoghq.com/api/v1/events"
```

Comment:
DataDog has released a v2 of this API recently -- should we support that instead?